src/go/rpk/pkg/config/manager.go

```diff
@@ -83,7 +83,11 @@ func (m *manager) FindOrGenerate(path string) (*Config, error) {
 			if err != nil {
 				return nil, err
 			}
-			conf.ConfigFile = m.v.ConfigFileUsed()
+			absPath, err := fp.Abs(m.v.ConfigFileUsed())
```

Comment:
ultra nit: Since all of these are in the manager, we could wrap that into a function and deduplicate some code.
Also if we were sure that of that conf is not going to be used we could just do:
```
conf.ConfigFile, err = GetAbs(...)
return conf, err
```

Reply:
So good. Adding it.